demos/Ollama Chat Service/whatExpress.html

```diff
@@ -0,0 +1,97 @@
+<html>
+<head>
+<title>Experimenting with Ollama (phi-3) in a Browser</title>
+<script>
+
+  const ollamaService = "http://localhost:11434";
+
+  // Handle click on "Ask" button
+  function askClicked(theButton) {
+    const single = "Answer with a single grammatically correct sentence."
+    console.debug("Button clicked");
+    // Empty out the response area
+    document.getElementById("phi3Output").innerText = "Working...";
```

Comment:
If the script is more flexible with any Ollama models, it would be better not having "phi3" in names such as "phi3Output".

Reply:
Thanks for catching that, @cindyli.  I made a similar change to the `ollama.html` web-app and its `ollama.js` script at some point, but didn't make the parallel change to `whatExpress.html`.